MarsTestAutomation/SpecFlow/SellerProfile.Certifications.feature

```diff
@@ -0,0 +1,85 @@
+Feature: SellerProfile_Certifications
+As a Seller
+I want to add Certification in my Profile Details
+So that
+The people looking my profile can see what Certification i done.
+	
+	
+	#	profile Certification Add
+	@sellerprofiletest
+	Scenario: Add Profile Certifications with data
+		Given I logged into Trade Skills portal successfully
+		And I click on Certifications and Add New button
+		When I Enter seller '<Certification>' , '<CertificationForm>', '<Year>' details and click on Add button
+		Then A popup should be shown with '<Message>' 
+
+		Examples: 
+         | Certification | CertificationForm | Year | Message                                      |
+         | Istqb         | Fundation level   | 2017 | Istqb has been added to your certification   |
+		 | English       | intership level   | 2015 | English has been added to your certification |
+		
+	@sellerprofiletest
+	Scenario: Add Profile Certifications withot data
+		Given I logged into Trade Skills portal successfully
+		And I click on Certifications and Add New button
```

Comment:
here you might want to click add as you are clicking the add button again please either move it in when and remove from given or remove the when and leave in given

Reply:
done